### ISSUE TITLE
I cannot delete files anymore

### ISSUE THREAD

**Issue by StanFromTraction** [files-community/Files#14465]
### Description

Whenever I try deleting a file or folder, even after running Files as admin, the administrator privilege prompt comes up, but clicking the CTA does nothing.
![image](https://github.com/files-community/Files/assets/105176736/567d4183-6fe5-4d69-8714-f424fbf50c2e)
![image](https://github.com/files-community/Files/assets/105176736/4e9d3c01-245d-4a78-b268-b75df530d4e3)

### Steps To Reproduce

1. Run Files
2. Try deleting something
3. Press the prompt's CTA
4. Nothing happens

### Requirements

- Fix bug that requires administrator privilege when it is already granted

### Files Version

3.1.1.0

### Windows Version

10.0.22631.2861

### Log File

[debug.log](https://github.com/files-community/Files/files/13940815/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report. What happens if you try in Windows File Explorer

**Comment by StanFromTraction** [comment]
You're right some weird things happened with the permissions on an OS level
